base_station/gui/src/components/AutonTask.vue

```diff
@@ -199,8 +231,14 @@ export default {
         {'topic': '/nav_status', 'type': 'NavStatus'},
         {'topic': '/gps', 'type': 'GPS'},
         {'topic': '/imu', 'type': 'IMU'},
-        {'topic': '/rr_drop_complete', 'type': 'RepeaterDrop'},
-        {'topic': '/debugMessage', 'type': 'DebugMessage'}
+        {'topic': '/rr_drop_complete', 'type': 'RepeaterDropComplete'},
```

Comment:
If we aren't using the repeater anymore, can we remove this?

Reply:
Similar to what Ashwin brought up, not sure if we're removing this from all code if it's just removed for this year?

Even if we do want to remove it however, my goal for this PR was just to give you guys what we have right now so that you can test it on Saturday in the integration test.